Mathlib/Topology/UniformSpace/UniformConvergenceTopology.lean

```diff
@@ -529,6 +529,15 @@ protected def uniformEquivPiComm : UniformEquiv (α →ᵤ ∀ i, δ i) (∀ i,
 #align uniform_fun.uniform_equiv_Pi_comm UniformFun.uniformEquivPiComm
 
 -- Like in the previous lemma, the diagram actually commutes by definition
+
```

Comment:
```suggestion

variable {α}

```
Otherwise alpha is explicit, which we don't want I think ?

Reply:
In other cases in this file, `α` is explicit whenever we have no assumptions that involve `α`.